pkg/model/stack.go

```diff
@@ -373,3 +368,11 @@ func GroupWarningsBySvc(fields []string) []string {
 	}
 	return result
 }
+
+func (s *Stack) DisplayWarnings() {
```

Comment:
could we keep this function on the  `cmd` package, since they are used to display info in the UI, and they are not part of the data model?